families/burrow_evm/src/sawtooth_burrow_evm/cli/create.go

```diff
@@ -71,7 +72,7 @@ func gen() (string, string, string, error) {
 	priv := sdk.GenPrivKey()
 	pub := sdk.GenPubKey(priv)
 	addr, err := common.PubToEvmAddr(pub)
-	return sdk.PrivToWif(priv), sdk.MustEncode(pub), addr.String(), err
+	return sdk.PrivToWif(priv), hex.EncodeToString(pub), addr.String(), err
```

Comment:
The files in families/burrow_evm/src/sawtooth_burrow_evm/cli/* are all going away, so this will have merge conflicts with https://github.com/hyperledger/sawtooth-core/pull/692